test/hotspot/jtreg/vmTestbase/nsk/jdi/ThreadReference/stop/stop002t.java

```diff
@@ -57,23 +60,131 @@ private int runIt(String args[]) {
         // as wrong parameter of JDI method ThreadReference.stop()
         stop002t stop002tNonThrowable = this;
 
+        // throwable object which will be used by debugger
+        // as valid parameter of JDI method ThreadReference.stop()
+        Throwable stop002tThrowable = new MyThrowable("Async exception");
+
         // Now the debuggee is ready for testing
         pipe.println(stop002.COMMAND_READY);
         String cmd = pipe.readln();
         if (cmd.equals(stop002.COMMAND_QUIT)) {
-            log.complain("Debuggee: exiting due to the command "
+            log.complain("Debuggee: premature debuggee exit due to the command "
                     + cmd);
-            return Consts.TEST_PASSED;
+            return Consts.TEST_FAILED;
+        }
+
+        /*
+         * TEST #2: async exception while suspended at a breakpoint.
+         */
```

Comment:
Where is a similar comment for TEST #1 ?

Reply:
Test #1 does not involve the debuggee since it is suppose to fail on the JDI side with an exception.

Reply:
> Test https://github.com/openjdk/jdk/pull/1 does not involve the debuggee
> since it is suppose to fail on the JDI side with an exception.

Still some comment is needed to explain this.
Otherwise, it is confusing that the subtests start from the TEST #2. :)

Reply:
Fixed.